src/screens/Survey/index.tsx

```diff
@@ -0,0 +1,26 @@
+import CustomButton from 'components/CustomButton';
+import CustomInput from './components/CustomInput';
+import { SURVEY } from './constants';
+import styles from './styles.module.scss';
+
+function Survey() {
+  return (
+    <>
+      <div className={styles.container}>
+        {SURVEY.map((form) => (
+          <form>
+            <div className={styles.topSection}>
+              <h2>{form.question}</h2>
+              <CustomInput inValue={form.inValue} isRadio={form.radio} />
+            </div>
+          </form>
```

Comment:
Be aware that you are creating an independent form for each item of the survey, instead of having one form that encapsulates all items in the `SURVEY` array